data/mapbox/fill_simple_outline.ts

```diff
@@ -0,0 +1,35 @@
+const fillSimpleFillOutline: any = {
+  version: 8,
+  name: 'Simple Fill With outline',
+  layers: [{
+    id: 'r0_sy0_st0',
+    type: 'fill',
+    paint: {
+      'fill-color': '#ff0000'
+    }
+  },
+  {
+    id: 'r0_sy0_st1',
+    type: 'line',
+    paint: {
+      'line-opacity': 0.5,
+      'line-color': '#00ff00',
+      'line-width': 2
+    },
+    layout: {
+      'line-cap': 'butt',
+      'line-join': 'round'
+    }
+  }],
+  metadata: {
+    geoStylerRef: {
```

Comment:
Mapbox suggests prefixing metadata, e.g. `mapbox:`. So here, instead of `geoStylerRef`, something like `geostyler:ref` could be used.

Reply:
Thanks @pprev94 and i refactored it here: [44db65d](https://github.com/geostyler/geostyler-mapbox-parser/pull/217/commits/44db65de8a2285ce2db754baa29b3076fab0e5f3)